sse2neon.h

```diff
@@ -805,6 +805,14 @@ FORCE_INLINE __m128d _mm_set_pd(double e1, double e0)
 #endif
 }
 
+// Broadcast double-precision (64-bit) floating-point value a to all elements of
+// dst.
+// https://software.intel.com/sites/landingpage/IntrinsicsGuide/#text=_mm_set_pd1
+FORCE_INLINE __m128d _mm_set_pd1(double a)
+{
+    return _mm_set_pd(a, a);
+}
+
```

Comment:
We have already implemented `_mm_set1_pd`.
I think you can define the macro like
```
#define _mm_set_pd1 _mm_set1_pd
```